src/mpid/ch4/src/ch4_init.c

```diff
@@ -455,6 +469,9 @@ int MPID_Init(int requested, int *provided)
     MPIR_Process.attrs.appnum = MPIR_Process.appnum;
     MPIR_Process.attrs.io = MPI_ANY_SOURCE;
 
+    /* Initialize comm hints */
```

Comment:
This comment doesn't provide more than the function name already says. Suggest remove.